errbot/backends/hipchat.py

```diff
@@ -94,8 +94,12 @@ def __init__(self, node=None, domain=None, resource=None, room=None, hipchat_use
             for k, v in hipchat_user.items():
                 setattr(self, k, v)
             # Quick fix to be able to all the parent.
-            node_domain, resource = hipchat_user['xmpp_jid'].split('/')
-            node, domain = node_domain.split('@')
+            if '/' in hipchat_user[u'xmpp_jid']:
```

Comment:
please remove the redundant u'  (they will be added back during the py2 conversion)